cmd/relayproxy/config/config.go

```diff
@@ -387,3 +413,26 @@ func loadArrayEnv(s string, v string, configMap map[string]interface{}) error {
 	}
 	return nil
 }
+
+func (c *Config) IsDebugEnabled() bool {
+	if c == nil {
+		return false
+	}
+	return c.LogLevel == "debug" || c.Debug
```

Comment:
Checking in lowercase to be ready for any configuration string.
```suggestion
	return strings.ToLower(c.LogLevel) == "debug" || c.Debug
```